src/app/server/Server.h

```diff
@@ -82,6 +82,10 @@
 #endif                                                       // CHIP_CONFIG_ENABLE_ICD_CIP
 #endif                                                       // CHIP_CONFIG_ENABLE_ICD_SERVER
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+#include <app/server/JointFabricDatastore.h>
```

Comment:
The include on line 36 should be removed?

Reply:
Agreed, and removed the include on line 36.